build/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/distroless/static:nonroot
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
Can't we use the ubi8-minimal image?

Reply:
We can't use ubi-minimal.. it doesn't have yum package manager installed.